lexicon/bpe.py

```diff
@@ -0,0 +1,142 @@
+__all__ = ["CreateBPELexiconJob"]
+
+import subprocess as sp
+import os
+import sys
+from typing import List, Optional, Set, Union
+import xml.etree.ElementTree as ET
+
+from sisyphus import Job, Task, tk, setup_path
+
+from i6_core.lib.lexicon import Lexicon, Lemma
+import i6_core.util as util
+
+
+class CreateBPELexiconJob(Job):
+    """
+    Create a Bliss lexicon from bpe transcriptions that can be used e.g, for lexicon constrained BPE search.
```

Comment:
> from bpe transcriptions

is that the correct term here? 

Isn't what the Job does rather
```suggestion
    In a Bliss lexicon replace the phonetic representation with a BPE decomposition of the words that can be used e.g, for lexicon constrained BPE search.
```